### PR TITLE
Start a dojo

### PR DESCRIPTION
**Pull request by heyitsmaimai**

### CODE REVIEW

**Comment by DanielBrierton** [comment]
Deploy preview for *cdf-website* ready!

Built with commit 2a5f4963950b3ebef9ed742448c5d0fc9a73dc0e

https://deploy-preview-7--cdf-website.netlify.com

File: components/index/StartADojo.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <h1>{{ $t('Want to start a Dojo?') }}</h1>
         <h2>{{ $t('It\'s just 3 steps!') }}</h2>
         <p>{{ $t('CoderDojo Champions are passionate, driven (not always techie!) volunteers who take charge of setting up, running and maintaining a Dojo.') }}</p>
-        <a href="#" class="c-button c-button--teal">{{ $t('Start a Dojo') }}</a>
+        <a href="/start-a-dojo" class="c-button c-button--teal">{{ $t('Start a Dojo') }}</a>
```
Comment (Wardormeur) [inline components/index/StartADojo.vue]: That's start-dojo, not start-a-dojo :)

File: components/startadojo/Hero.vue [hunk 2]
```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="c-hero">
+    <h1 class="c-hero__header1">{{ $t('Let\'s start a Dojo to help engage young people near you') }}</h1>
+    <div class="c-hero__step-cards">
+      <div class="c-hero__step-card">
+        <img class="c-hero__image" src="~/static/images/doc.svg" />
+        <h1 class="c-hero__step-header c-hero__orange">{{ $t('Step 1') }}</h1>
+        <p class="c-hero__step-text"><span class="c-hero__step-text-first-word c-hero__orange">{{ $t('Register ') }}</span>{{ $t('with us as a Champion so we can support you as you build your club.') }}</p>
```
Comment (Wardormeur) [inline components/startadojo/Hero.vue]: That doesn't work for translations. 
I would recommend using js to do that instead (translate the whole sentence then highlight on created the specific keyword)

File: components/startadojo/Hero.vue [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      </div>
+      <div class="c-hero__step-card">
+        <img class="c-hero__image" src="~/static/images/pin.svg" />
+        <h1 class="c-hero__step-header c-hero__purple">{{ $t('Step 2') }}</h1>
+        <p class="c-hero__step-text"><span class="c-hero__step-text-first-word c-hero__purple">{{ $t('Find ') }}</span>{{ $t('a public location to hold your Dojo. Libraries, schools, businesses and clubs are often used.') }}</p>
```
Comment (Wardormeur) [inline components/startadojo/Hero.vue]: same as ^

File: components/startadojo/Hero.vue [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+      </div>
+      <div class="c-hero__step-card">
+        <img class="c-hero__image" src="~/static/images/screen.svg" />
+        <h1 class="c-hero__step-header c-hero__teal">{{ $t('Step 3') }}</h1>
+        <p class="c-hero__step-text"><span class="c-hero__step-text-first-word c-hero__teal">{{ $t('Create ') }}</span>{{ $t('an event using our events system and use over 200 ready made projects to run a great first event.') }}</p>
```
Comment (Wardormeur) [inline components/startadojo/Hero.vue]: same as ^^

File: components/startadojo/Hero.vue [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+      </div>      
+    </div>
+     <div class="c-hero__cta">
+      <a href="#" class="c-button c-button--teal">{{ $t('Start a Dojo') }}</a>
```
Comment (Wardormeur) [inline components/startadojo/Hero.vue]: /start-dojo :)

File: components/startadojo/Hero.vue (hunk 2 above)
Comment (heyitsmaimai) [reply components/startadojo/Hero.vue]: dont

File: components/startadojo/Hero.vue [hunk 6]
```diff
@@ -3,19 +3,28 @@
     <h1 class="c-hero__header1">{{ $t('Let\'s start a Dojo to help engage young people near you') }}</h1>
     <div class="c-hero__step-cards">
       <div class="c-hero__step-card">
-        <img class="c-hero__image" src="~/static/images/doc.svg" />
-        <h1 class="c-hero__step-header c-hero__orange">{{ $t('Step 1') }}</h1>
-        <p class="c-hero__step-text"><span class="c-hero__step-text-first-word c-hero__orange">{{ $t('Register ') }}</span>{{ $t('with us as a Champion so we can support you as you build your club.') }}</p>
+        <img class="c-hero__step-image" src="~/static/images/doc.svg" />
+        <h1 class="c-hero__step-header">
+          <span class="c-hero__step-number">{{ $t('1') }}</span>
+          <span class="c-hero__step-name">{{ $t('Register') }}</span>
```
Comment (Wardormeur) [inline components/startadojo/Hero.vue]: Still doesn't work great for LTR, but considering nothing as been prepared for that, it can prob wait

**Pull request merged**